test/unittests/instantiate_test.cpp

```diff
@@ -0,0 +1,72 @@
+#include "execute.hpp"
+#include "utils.hpp"
+#include <gtest/gtest.h>
+
+using namespace fizzy;
+
+namespace
+{
+constexpr unsigned PageSize = 65536;
```

Comment:
```suggestion
constexpr unsigned page_size = 65536;
```

I would prefer to rename this consistently in the source if we decide so.